subworkflows/local/bam_variant_calling_germline_all/main.nf

```diff
@@ -191,6 +193,18 @@ workflow BAM_VARIANT_CALLING_GERMLINE_ALL {
         versions = versions.mix(BAM_VARIANT_CALLING_GERMLINE_MANTA.out.versions)
     }
 
+    // DYSGU
+    if (tools.split(',').contains('dysgu')) {
+        BAM_VARIANT_CALLING_DYSGU (
+            cram,
+            fasta,
+            fasta_fai
```

Comment:
you need to add a fourth input channels with intervals. It is probably just called `intervals`.